packages/core/src/components/MenuButton/__stories__/MenuButton.stories.tsx

```diff
@@ -92,11 +105,16 @@ export const DifferentIcon = {
       </Menu>
     </MenuButton>
   ),
-
-  name: "Different Icon"
```

Comment:
Storybook uses lodash's `startCase` method:
https://lodash.com/docs/#startCase
so ```export const DifferentIcon``` would result in a story with the name "Different Icon", so no need for the name here as it would result the same